docs/application/dotnet/guides/user-interface/nui/videoview.md

```diff
@@ -16,27 +16,33 @@ The following basic example shows how to create a `VideoView` object:
 
 **Figure: Basic VideoView**
 
-![Basic VideoView](./media/basicVideoView.png)
+![Basic VideoView](../screenshots/VideoView.png)
 
-1.  To use the `VideoView` class, add the following namespaces:
+1.  To use the `VideoView` class, add the following namespace and assembly:
 
-    ```csharp
-    using Tizen.NUI;
-    using Tizen.NUI.BaseComponents;
-    ```	    
+    ```xaml
+    xmlns:base="clr-namespace:Tizen.NUI.BaseComponents;assembly=Tizen.NUI"
+    ```
   
 2.   The video file is assumed to be in the resources directory. Create an instance of the `VideoView` class and use the `ResourceUrl` property to pass the path to the video file. Set `WidthResizePolicy` to make `VideoView` instance use full width of the window and set `HeightResizePolicy` to maintain aspect ratio of video. And then you can invoke the `Play()` method to start video:
 
-      ```csharp
-      VideoView player = new VideoView();
-      player.ResourceUrl = DirectoryInfo.Resource + "sample.mp4";
-      player.WidthResizePolicy = ResizePolicyType.FillToParent;
-      player.HeightResizePolicy = ResizePolicyType.DimensionDependency;
-      player.Play();
+      ```xaml
+      <base:VideoView x:Name="player" ResourceUrl="*Resource*/images/sample.3gp" 
+                    WidthSpecification="720" HeightSpecification="405"
+                    Underlay="False"  />
+      <comp:Button x:Name="start_btn" Text="Start Video" Position="0,800" WidthSpecification="-1"
+                    HeightSpecification="100" Clicked="start_Clicked"/>
+      ```
 
-      Window.Instance.Add(player);
+      ```csharp
+      private void start_Clicked(object sender, ClickedEventArgs e)
+        {
+            player.Play();
+        }
       ```
 
+![Basic VideoView](../media/basicVideoView.png)
+
 > [!NOTE]  
 > You can set the video file to be played in `new VideoView()`, or by modifying the `Video` property.
```

Comment:
Line 47: "You can set the video file to be played in `new VideoView()`, or by modifying the `Video` property."
Seeking clarification: “`new VideoView()`”, is “new” being used here as an instance of the class or should we use it separately?
```suggestion
> You can set the video file to be played in a new `VideoView()`, or by modifying the `Video` property.
```